selfdrive/loggerd/logger.h

```diff
@@ -5,7 +5,13 @@
 #include <stdint.h>
 #include <pthread.h>
 #include <bzlib.h>
-
+#include <kj/array.h>
+#include <capnp/serialize.h>
+#if defined(QCOM) || defined(QCOM2)
```

Comment:
add some whitespace here

Reply:
let's also just make logger.c into a cc file so we can remove the extern c stuff